### PR TITLE
New version: VoiceAttack.VoiceAttack version 1.12

### PR DESCRIPTION
**Pull request by michaelabon**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138417)

### CODE REVIEW

**Comment by michaelabon** [comment]
@microsoft-github-policy-service agree

File: manifests/v/VoiceAttack/VoiceAttack/1.12/VoiceAttack.VoiceAttack.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+PackageIdentifier: VoiceAttack.VoiceAttack
+PackageVersion: "1.12"
+InstallerType: inno
+Installers:
+- Architecture: x86
+  InstallerUrl: https://www.voiceattack.com/filesend.aspx?id=voiceattackinstaller_112.exe
+  InstallerSha256: F2092CD0D32B2F64742BFE65A30DBF2658FEB293D7A596DF94E6B78689937648
+ManifestType: installer
+ManifestVersion: 1.5.0
```
Comment (michaelabon) [inline manifests/v/VoiceAttack/VoiceAttack/1.12/VoiceAttack.VoiceAttack.installer.yaml:12]: Validated using the 1.6.0 JSON Schema that the rest of this manifest conforms:

```suggestion
ManifestVersion: 1.6.0
```

File: manifests/v/VoiceAttack/VoiceAttack/1.12/VoiceAttack.VoiceAttack.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+PackageIdentifier: VoiceAttack.VoiceAttack
+PackageVersion: "1.12"
+PackageLocale: en-US
+Publisher: VoiceAttack.com
+PackageName: VoiceAttack
+License: Copyright © 2008-2020 VoiceAttack.com
+ShortDescription: Voice-activated control for your PC games and apps.
+ManifestType: defaultLocale
+ManifestVersion: 1.5.0
```
Comment (michaelabon) [inline manifests/v/VoiceAttack/VoiceAttack/1.12/VoiceAttack.VoiceAttack.locale.en-US.yaml:12]: Validated using the 1.6.0 JSON Schema that the rest of this manifest conforms:

```suggestion
ManifestVersion: 1.6.0
```

File: manifests/v/VoiceAttack/VoiceAttack/1.12/VoiceAttack.VoiceAttack.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: VoiceAttack.VoiceAttack
+PackageVersion: "1.12"
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.5.0
```
Comment (michaelabon) [inline manifests/v/VoiceAttack/VoiceAttack/1.12/VoiceAttack.VoiceAttack.yaml:8]: Validated using the 1.6.0 JSON Schema that the rest of this manifest conforms:

```suggestion
ManifestVersion: 1.6.0
```

**Review by michaelabon — COMMENTED** [review]
wingetcreate used the 1.5.0 schema, but this also passes the 1.6.0 schema validation. I'm updating the `ManifestVersion` keys to reflect that.

**Comment by stephengillie** [comment]
Hi @michaelabon,

> This PR's version number 1.12 has 2 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 1.10.3, which has 3 parameters.

Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)

(Automated response - build 717)

**Comment by michaelabon** [comment]
> Hi @michaelabon,
> 
> > This PR's version number 1.12 has 2 parameters (sets of numbers between dots - major, minor, etc), which is less than the current manifest's version 1.10.3, which has 3 parameters.
> 
> Does this PR's version number **exactly** match the version reported in the Apps & features Settings page? (Feel free to attach a screenshot.)
> 

> (Automated response - build 717)

Yes, this app is reported as just the major.minor version of 1.12.

Screenshot of the Windows Settings window, under the page App, Installed Apps. There is a search box for the phrase "VoiceAttack". There is 1 app found, which is listed as VoiceAttack version 1.12

Within the app, it refers to itself as just version 1.12

Screenshotof the Voice Attack app's About page. The heading is "A few details about your setup". The text is "VoiceAttack 64-bit version: v1.12"

**Review by stephengillie — APPROVED** [review]

**Pull request merged**